packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -153,4 +170,21 @@ export class OfflineQueueLink extends ApolloLink {
       });
     }
   }
+
+  private updateIds(
+    operationEntry: OperationQueueEntry,
+    result: FetchResult
+  ): void {
+    const operation = operationEntry.operation;
+    const optimisticResponse = operationEntry.optimisticResponse;
+    if (optimisticResponse && optimisticResponse[operation.operationName].id.startsWith("client:")) {
```

Comment:
Bad habbit of leaking implementations. We have two dangling flags in the code: 
`client:` is in 2 classes. Let's make separate method in optimisticResponse file and use it here.

Reply:
should be fixed now